.github/workflows/cd.yml

```diff
@@ -53,15 +53,14 @@ jobs:
           $WebClient.DownloadFile("https://github.com/deluge-torrent/gvsbuild-release/releases/download/latest/gvsbuild-py${{ matrix.python }}-vs16-${{ matrix.arch }}.zip","C:\GTK.zip")
           7z x C:\GTK.zip -oc:\GTK
           $env:Path = "C:\GTK\release;C:\GTK\release\bin;C:\GTK\release\lib;$env:Path"
-          python -m pip install --upgrade pip wheel
+          python -m pip install --upgrade pip
```

Comment:
wheel should be kept here so it's available when pip installs packages

we should either go for `python -m pip install` or use `pip install` (seems fine to use this) but let's try not to mix usage. As CirnoT mentioned it's a bad practice that under other environments can give you unexpected results.